pkg/sentry/loader/loader.go

```diff
@@ -392,6 +392,13 @@ func Load(ctx context.Context, args LoadArgs, extraAuxv []arch.AuxEntry, vdso *V
 	m.SetAuxv(auxv)
 	m.SetExecutable(d)
 
+	symbolValue, err := getSymbolValueFromVDSO("rt_sigreturn")
```

Comment:
should we report an error here if we fail to find rt_sigreturn?

Reply:
Thre is no 'rt_sigreturn' on x86 platform. 
We can't return an error, otherwise the platform won't get up.
For your reference:
On x86:
 nm vdso.so
ffffffffff701030 W clock_gettime
ffffffffff700480 d _DYNAMIC
ffffffffff701100 W getcpu
ffffffffff701070 W gettimeofday
                 U _GLOBAL_OFFSET_TABLE_
0000000000000000 A LINUX_2.6
ffffffffff6ff000 a _params
ffffffffff7010d0 W time
ffffffffff701030 T __vdso_clock_gettime
ffffffffff701100 T __vdso_getcpu
ffffffffff701070 T __vdso_gettimeofday
ffffffffff700000 a VDSO_PRELINK
ffffffffff7010d0 T __vdso_time
ffffffffff701110 t _ZN4vdso13ClockRealtimeEP8timespec
ffffffffff7011d0 t _ZN4vdso14ClockMonotonicEP8timespec

On Arm64:
nm vdso.so
ffffffffff7003e8 d _DYNAMIC
ffffffffff7010e8 T __kernel_clock_getres
ffffffffff701038 T __kernel_clock_gettime
ffffffffff701070 T __kernel_gettimeofday
ffffffffff701120 T __kernel_rt_sigreturn
0000000000000000 A LINUX_2.6.39
ffffffffff6ff000 a _params
ffffffffff700000 a VDSO_PRELINK
ffffffffff701130 t _ZN4vdso13ClockRealtimeEP8timespec
ffffffffff7011e8 t _ZN4vdso14ClockMonotonicEP8timespec

Reply:
we have to add rt_sigreturn for x86.

Reply:
Done.